browser/lib/util/http.ts

```diff
@@ -0,0 +1,176 @@
+import * as Utils from '../../../common/lib/util/utils';
+import Defaults from '../../../common/lib/util/defaults';
+import ErrorInfo from '../../../common/lib/types/errorinfo';
+import { IHttp, RequestCallback, RequestParams } from '../../../common/types/http';
+import HttpMethods from '../../../common/constants/HttpMethods';
+
+type Rest = any;
```

Comment:
I'm guessing that this and `Realtime` below it, both repeated in the `nodejs` variant, represent some kind of temporary placeholder - i.e. they'll be replaced at some point later, presumably when these types get formal TypeScript-aware definitions?

If that is the case then I think best practice here would be to add comments to these type definitions, individually:

- stating that they are temporary
- explicitly mentioning `TODO` as a marker in the code
- including a link to the GitHub issue which you will work on to fix them

Reply:
Owen's resolution (addition of comments) looks acceptable to me, given [his explanation](https://github.com/ably/ably-js/pull/800#discussion_r723320445), however I would rather leave it to @SimonWoolf to resolve this conversation in order to get his acknowledgement that this is satisfactory for now.